roles/maintainerr/defaults/main.yml

```diff
@@ -0,0 +1,178 @@
+
+################################
+# Traefik
+################################
+
+maintainerr_traefik_sso_middleware: "{{ traefik_default_sso_middleware }}"
+maintainerr_traefik_middleware_default: "{{ traefik_default_middleware
+                                          + (',themepark-' + lookup('vars', maintainerr_name + '_name', default=maintainerr_name)
+                                            if (maintainerr_themepark_enabled and global_themepark_plugin_enabled)
+                                            else '') }}"
+maintainerr_traefik_middleware_custom: ""
+maintainerr_traefik_certresolver: "{{ traefik_default_certresolver }}"
+maintainerr_traefik_enabled: true
+maintainerr_traefik_api_enabled: false
+maintainerr_traefik_api_endpoint: ""
+
+################################
+# THEME
+################################
+
+# Options can be found at https://github.com/themepark-dev/theme.park
+maintainerr_themepark_enabled: false
+maintainerr_themepark_app: "maintainerr"
+maintainerr_themepark_theme: "{{ global_themepark_theme }}"
+maintainerr_themepark_domain: "{{ global_themepark_domain }}"
+maintainerr_themepark_addons: []
+
+################################
+# Docker
+################################
+
+# Container
+maintainerr_docker_container: "{{ maintainerr_name }}"
+
+# Image
+maintainerr_docker_image_pull: true
+maintainerr_docker_image_repo: "jorenn92/maintainerr"
+maintainerr_docker_image_tag: "latest"
+maintainerr_docker_image: "{{ lookup('vars', maintainerr_name + '_docker_image_repo', default=maintainerr_docker_image_repo)
+                            + ':' + lookup('vars', maintainerr_name + '_docker_image_tag', default=maintainerr_docker_image_tag) }}"
+
+# Ports
+maintainerr_docker_ports_defaults: []
+maintainerr_docker_ports_custom: []
+maintainerr_docker_ports: "{{ lookup('vars', maintainerr_name + '_docker_ports_defaults', default=maintainerr_docker_ports_defaults)
+                            + lookup('vars', maintainerr_name + '_docker_ports_custom', default=maintainerr_docker_ports_custom) }}"
+
+# Envs
+maintainerr_docker_envs_default:
+  UMASK: "002"
+  TZ: "{{ tz }}"
+  LOG_LEVEL: "{{ maintainerr_log_level }}"
+maintainerr_docker_envs_custom: {}
+maintainerr_docker_envs: "{{ lookup('vars', maintainerr_name + '_docker_envs_default', default=maintainerr_docker_envs_default)
+                           | combine(lookup('vars', maintainerr_name + '_docker_envs_custom', default=maintainerr_docker_envs_custom)) }}"
+
+# Commands
+maintainerr_docker_commands_default: []
+maintainerr_docker_commands_custom: []
+maintainerr_docker_commands: "{{ lookup('vars', maintainerr_name + '_docker_commands_default', default=maintainerr_docker_commands_default)
+                               + lookup('vars', maintainerr_name + '_docker_commands_custom', default=maintainerr_docker_commands_custom) }}"
+
+# Volumes
+maintainerr_docker_volumes_default:
+  - "{{ maintainerr_paths_location }}:/app/config"
```

Comment:
shouldn't the path be `/opt/data`?

Reply:
Looks that way to me from the installation docs.

Reply:
You're talking about here?

https://docs.maintainerr.info/Installation/#linux-permissions-example